code/src/CSS/MovieDetails.css

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Nice approach to have the .css separated for different components. It makes it more adjustable and easier to edit.